.github/workflows/publish-extensions.yml

```diff
@@ -43,19 +54,28 @@ jobs:
       - name: Sign Production Firefox version
         continue-on-error: true
         run: yarn web-ext sign --channel=unlisted
+        # run: yarn web-ext sign --channel=listed
         env:
-          WEB_EXT_API_KEY: ${{ secrets.WEB_EXT_API_KEY }}
-          WEB_EXT_API_SECRET: ${{ secrets.WEB_EXT_API_SECRET }}
-      - uses: actions/upload-artifact@v2
-        name: Upload Artifact - Firefox Add-On XPI
-        with:
-          name: connect-addon
-          path: web-ext-artifacts/*.xpi
+          WEB_EXT_API_KEY: ${{ secrets.FIREFOX_API_KEY }}
+          WEB_EXT_API_SECRET: ${{ secrets.FIREFOX_API_SECRET }}
       - name: Get Firefox addon filename
         id: addon-file
         run: echo "::set-output name=addon_path::$(find ./web-ext-artifacts -type f -iname "*.xpi" | tail -n1)"
       - name: Rename Firefox addon file
         run: cp ${{ steps.addon-file.outputs.addon_path }} stacks-wallet-firefox.xpi
+      - name: Upload Artifact - Firefox Add-On XPI
+        uses: actions/upload-artifact@v2
+        with:
+          name: stacks-wallet-firefox
+          path: stacks-wallet-firefox.xpi
+      - name: Upload addon to Firefox store
```

Comment:
This whole action is redundant - and won't work if we use it in combination with the `yarn web-ext sign` step. I'd say let's just use the existing (above) line, which we already know works.

Reply:
oh, the wording in the issue made me think auto-publishing wasn't happening for chrome nor firefox. So it seems it only wasn't happening for chrome? I was under the impression the `yarn web-ext sign` step wasn't publishing anything, but just signing the package so it can be published. This step was added to do the actual publishing.

If `yarn web-ext sign` publishes the package (sans release notes and source code uploads) I'll remove this step.

Reply:
Yeah, it's confusing. It's not possible to "publish" to Firefox automatically. But "sign" actually uploads the version, which you then have to manually add some details to in order to finally publish it